client/my-sites/patterns/components/pattern-gallery/client.tsx

```diff
@@ -161,6 +163,16 @@ export const PatternGalleryClient: PatternGalleryFC = ( props ) => {
 					enableMasonry={ isGridView && isPageLayouts }
 					itemSelector=".pattern-preview"
 				>
+					{ displayPlaceholder && (
+						<PatternPreviewPlaceholder
+							className={ classNames( {
+								'pattern-preview--grid': isGridView,
+								'pattern-preview--list': ! isGridView,
+							} ) }
+							title="…"
```

Comment:
<img width="346" alt="Screenshot 2024-04-15 at 11 18 19" src="https://github.com/Automattic/wp-calypso/assets/5598437/898965b2-d3c1-44ee-9934-d816c927f96a">
Maybe it's personal, but extra ellipsis looks like "something broken". I would just don't show title.

Reply:
I agree it's better to skip that ellipsis. I've added a `Copy` button instead and made the title empty.